package.json

```diff
@@ -4,7 +4,11 @@
   "description": "Website build it with DOM manipulation",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "dev": "webpack --mode development",
+    "build": "webpack --mode production",
+    "start": "webpack-dev-server --mode development --open",
```

Comment:
Add the `web-dev-server` package to your `package.json` file to enable the `npm run start` command to run. 
 ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/devdep_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/))

Reply:
- [ ] As pointed out earlier, the script is still producing some errors when the user runs the `npm start` command. Do you mind fixing this, please?